vobject/base.py

```diff
@@ -296,7 +299,12 @@ def updateTable(x):
             qp = True
             self.singletonparams.remove('QUOTED-PRINTABLE')
         if qp:
-            self.value = self.value.decode('quoted-printable')
+            if six.PY3:
+                charset = self.params.get('CHARSET', ['UTF-8'])[0]
+                self.value = quopri.decodestring(self.value).decode(charset)
```

Comment:
I'd like to see some comment on what's happening here... If we're in a py3 env, why the need to decode at all?